opcua_dbus.c

```diff
@@ -192,9 +233,182 @@ bool dbus_temp_unpack_signal(GVariant *parameters, uint32_t *subscription_id, do
     return true;
 }
 
-void dbus_connect_g_signal(GCallback func)
+void dbus_connect_temp_g_signal(GCallback func)
 {
-    assert(NULL != dbusproxy);
+    assert(NULL != dbusproxy_temp);
     assert(NULL != func);
-    g_signal_connect(dbusproxy, "g-signal", func, NULL);
+    g_signal_connect(dbusproxy_temp, "g-signal", func, NULL);
+}
+
+void dbus_connect_ports_g_signal(GCallback func)
+{
+    assert(NULL != dbusproxy_ports);
+    assert(NULL != func);
+    g_signal_connect(dbusproxy_ports, "g-signal", func, NULL);
```

Comment:
Here I'd prefer one function instead of a duplicate with just the parameters changed. Please add the dbusproxy as an inparameter and let it be a single function.